src/components/heatmapModeForm/HeatmapModeForm.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from "react";
+import Form from "./../../components/form/Form";
+import FormField from "grommet/components/FormField";
+import FormFields from "grommet/components/FormFields";
+import CheckBox from "grommet/components/CheckBox";
+import type { ConfigObject } from "../../types/Heatmap";
+import { inputHandler } from "../../utils/InputHandler";
+import type { allInputTypes } from "../../utils/InputHandler";
+
+type Props = {
+	configObject: ConfigObject,
+	onChange: (string, allInputTypes) => void
+};
+
+export default function HeatmapModeForm({
+	configObject,
+	onSubmit,
+	onChange
+}: Props) {
+	return (
+		<Form heading="Modi aktivieren" onSubmit={() => {}}>
```

Comment:
Doesn't require a submit button. Make the `onSubmit` callback optional in the `<Form />` component and make the button invisible, if there is no callback available.